vtr_flow/arch/multi_die/aman_3d_limited.xml

```diff
@@ -0,0 +1,1579 @@
+<architecture>
```

Comment:
Add comment to top of architecture. Where did it come from, what's it for, any caveats?
Add a reference to the fpt paper as it definitely has info on the limited out put architecture, and results for it.
Rename to something other than aman (describe what it is, not who wrote it).
Does his have delays between dice? If not, update it to have some reasonable switch delay between dice.

Reply:
I have to fix a bunch of 3D architectures and add to them. Also, in another branch, I am working on developing some more tests on 3D architecture too. These comments are added on that branch.